File: file_scripts/file_memory.py
import os
from pathlib import Path




def count_size(file_name:str) -> int:
    try:
        filepath = Path(file_name)
        if filepath.exists():
            if filepath.is_file():
                return round(int(filepath.stat().st_size) / (1024 * 1024))
            else:
                raise TypeError("This is not a file")
        else:
            raise KeyError("File not found")                
    except Exception as e:
        raise Exception(f"Error : {e}")

File: file_scripts/file_memory.cpp
#include <iostream>
#include <string>
#include <algorithm>
#include <ctime>
#include <filesystem>

using namespace std;
namespace fs = std::filesystem;

uintmax_t count_size(const std::string filename){
   
    try{
        if (fs::exists(filename)){
            uintmax_t size = fs::file_size(filename);
            return size;
        }else{
            std::cout << "File not found" << endl;
            return NULL;
        }
    }catch(std::exception& e){
        std::cerr << "Error: " << e.what() << endl;
    }
}


int main(){
    uintmax_t res = count_size("test.txt");
    std::cout << res << endl; // baits
    return 0;
}